doc/content/source/postprocessors/PeriodicTimeIntegratedPostprocessor.md

```diff
@@ -0,0 +1,51 @@
+# PeriodicTimeIntegratedPostprocessor
+
+This Postprocess provides the same functionationlity as [MultipliedTimeIntegratedPostprocessor.md] except this integrals value will periodically be set to 0 in order to integrate values over a single period.
+
+\begin{equation}
+  I \approx
+  C
+  \int_0^t
+  f(t) dt
+  \:\:
+  \text{mod}
+  \left(
+    \frac{1}{f}
+  \right)
+\end{equation}
+
+Where $f$ is the cycle frequency specified by the parameter `cycle_frequency`
+
+
+# Example Integration
+
+
+!media media/periodic_integral.png
+       style=width:50%;display:block;margin-top:3em;margin-left:auto;margin-right:auto;
```

Comment:
Since this image is not the result of the included test explain, I would recommend removing the image and this section. Personal, I think this image and the corresponding input file could be used as part of a power regulation tutorial. Also, since this in a periodic integrator, I believe it would be more beneficial for the x-axis to be the number of periods, instead of overall time. 

If you want to keep the image here, I would recommend elaborating on the condition of the test parameters and include the input file in the testing folder with a syntax test. 

```suggestion
```

Reply:
I agree that this would probably be more appropriately placed into a tutorial for the power regulator once I have developed that.